data/assets/07_matrix_view.css

```diff
@@ -3,29 +3,50 @@
   color: inherit;
 }
 
+.div-data {
+  width: 100%;
+  height: 100%;
+}
+
 .td-data {
   font-size: medium;
+  border:1px solid black;
+  border-collapse: collapse;
 }
 
 .td-diagonal {
-    background-color:  white
+  background-color:  white
+  border:1px solid black;
+  border-collapse: collapse;
 }
 
-.td-agent {
+.td-from-agent {
   font-weight: bold;
   background-color: powderblue;
   text-align: left;
-  white-space: nowrap;
+  border:1px solid black;
+  border-collapse: collapse;
 }
 
-.vertical {
-writing-mode: vertical-lr;
-min-width: 18px;
+.td-to-agent {
+  font-weight: bold;
+  background-color: powderblue;
```

Comment:
The background color should be configured via a variable in `01.css`, so that things can be "themed" at one place. ... and I also don't like the `powderblue` :-(.

Reply:
OK, will use the the light gray "--gray-1" from 01.css for background color

Reply:
Updated in PR#66